### ISSUE TITLE
Add accessiblity linting

### ISSUE THREAD

**Issue by tskarhed** [grafana/grafana#37782]
We should consider using this plugin for eslint

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y

We might have a lot of issues, which means we should have a systematic approach for implementing this.

**Comment by tharun208** [comment]
@tskarhed I like to work on this. I will raise a draft pr soon.

**Comment by tskarhed** [comment]
> @tskarhed I like to work on this. I will raise a draft pr soon.

I think we have to be a bit systematic about it, since it will throw a bunch of errors as soon as we enable it.

**Comment by osg-grafana** [comment]
cc @osg-grafana
